enterprise_gateway/services/sessions/kernelsessionmanager.py

```diff
@@ -30,8 +30,15 @@ class KernelSessionManager(LoggingConfigurable):
         state consistent.
     """
 
-    enable_persistence = Bool(default_value=False, config=True,
-        help="""Enable kernel session persistence.  Default = False""")
+    # Session Persistence
+    session_persistence_env = 'EG_SESSION_PERSISTENCE'
```

Comment:
I'd like to include `KERNEL` in the config option's env variable so that it is clear this only pertains to _kernel sessions_ and not the more common _notebook sessions_.  This difference is conveyed in the command line option due to the class prefix of `KernelSessionManager`.  I think using `EG_KERNEL_SESSION_PERSISTENCE` is appropriate.

Reply:
Sure thing.